things/api.py

```diff
@@ -86,16 +85,26 @@ def tasks(uuid=None, include_items=False, **kwargs):  # noqa: C901
         - `tag == True`, only include tasks _with_ tags.
         - `tag == None` (default), then include all tasks.
 
-    start_date : bool or None, optional
+    start_date : bool, str or None, optional
         - `start_date == False`, only include tasks _without_ a start date.
         - `start_date == True`, only include tasks _with_ a start date.
+        - `start_date == 'future'`, only include tasks with a future start date.
+        - `start_date == 'past'`, only include tasks with a past start date.
         - `start_date == None` (default), then include all tasks.
 
-    deadline : bool or None, optional
+    deadline : bool, str or None, optional
         - `deadline == False`, only include tasks _without_ a deadline.
         - `deadline == True`, only include tasks _with_ a deadline.
+        - `deadline == 'future'`, only include tasks with a deadline in the future.
+        - `deadline == 'past'`, only include tasks with a deadline in the past.
         - `deadline == None` (default), then include all tasks.
 
+    deadline_suppressed : bool or None, optional
+        - `deadline_suppressed == True`, only include tasks with a overdue deadline
+          that have been moved from the today view (Inbox, Anytime, Someday).
+        - `deadline_suppressed == False`, only include tasks with an overdue deadline
+          that have not been moved from the today view to another view (Inbox, Anytime, Someday).
```

Comment:
Are only tasks with overdue deadlines included or all tasks no matter if their deadline is overdue or not?